platform/ios/test/MGLTileSetTests.mm

```diff
@@ -43,7 +44,7 @@ - (void)testMBGLTileSet {
     XCTAssertEqual([tileSet mbglTileset].zoomRange.max, 2);
     
     // when the tile set has an attribution
-    tileSet.attribution = @"my tileset";
+    tileSet.attribution = @"my tileset ©";
```

Comment:
© is a decent test, since it falls outside of ASCII. In #5999, I used ©️, which pairs © with a variation selector for an emoji look. But something outside the Basic Multilingual Plane, for example in the [Miscellaneous Symbols and Pictographs](https://en.wikipedia.org/wiki/Miscellaneous_Symbols_and_Pictographs) block, would be even better for detecting mojibake.